changelog.d/6910.bugfix

```diff
@@ -0,0 +1 @@
+Fixed set a user as an admin with the new API.
```

Comment:
which new api? can you be more specific about which APIs are being fixed please?